components/Common/Button/index.module.scss

```diff
@@ -0,0 +1,85 @@
+.button {
+  @apply px-[18px]
+  py-[10px]
```

Comment:
If you see on our example, we add a space after the first line containing `@apply` to kinda hint that these tokens are part of the apply statement (on my review comment I also kinda mentioned that)

Could you make this final change? 🙇